app/javascript/vue/shared/tinymce.vue

```diff
@@ -43,7 +44,7 @@
                     :name="fieldName"
                     aria-hidden="true">
           </textarea>
-          <input type="hidden" class="tiny-mce-images" name="tiny_mce_images" value="[]">
+          <input type="hidden" id="tiny-mce-images" name="tiny_mce_images" value="[]">
```

Comment:
I think better fix JS, which can be multiple inputs on the task. If you load few editors